src/Catalyst.TestUtils/ProtocolHelpers/PublicEntryHelper.cs

```diff
@@ -38,11 +40,13 @@ public static class PublicEntryHelper
         public static PublicEntry GetPublicEntry()
         {
             var amount = new Random().Next(78588446).ToByteArray(new Bytes.Endianness());
+            var fees = new Random().Next(78588446).ToByteArray(new Bytes.Endianness());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'fees'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=5051852)